crud/tenant_crud.py

```
@@ -0,0 +1,135 @@
+    return db.query(models.Tenant).offset(skip).limit(limit).all()
+
+def update_tenant(db: Session, tenant_id: str, tenant_data: schemas.TenantUpdateSchema, conteudo_loja: Optional[str] = None, menu_image_url: Optional[str] = None):
+    from .personality_crud import create_personality, get_personality_by_name # Importação local
+
+    db_tenant = db.query(models.Tenant).filter(models.Tenant.tenant_id == tenant_id).first()
+    if not db_tenant:
+        return None
+    
+    update_data = tenant_data.model_dump(exclude_unset=True)
+    
+    if "nome_loja" in update_data: db_tenant.nome_loja = update_data["nome_loja"]
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)